lib/generators/active_record/devise_security_generator.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails/generators/active_record'
+require 'generators/devise/orm_helpers'
+
+module ActiveRecord
+  module Generators
+    class DeviseSecurityGenerator < Base
+      source_root File.expand_path('templates', __dir__)
+
+      def copy_migration_file
+        migration_template 'session_histories.rb.erb',
+                           "db/migrate/create_#{table_name}.rb",
+                           migration_version: migration_version,
+                           ip_column: ip_column
+      end
+
+      def ip_column
+        inet? ? 'inet' : 'string'
+      end
+
+      def inet?
```

Comment:
Can you elaborate on this one? Normally I'm not a big fan on one-liner pass through methods but I'm interested in your reasoning here.

Reply:
Actually, I just preferred to use one-liner for simple if conditions.

Reply:
Hopefully, we can fully enable the rubocop in this gem to have a standard code style.